modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -857,6 +859,15 @@ private void _addObjectDefinitionResources(
 						postObjectDefinitionObjectRelationship(
 							objectDefinitionId, objectRelationship);
 
+				_objectRelationshipLocalService.updateEdge(
+					objectRelationshipEdge, objectRelationship.getId());
+
+				ObjectField relationshipObjectField =
+					objectRelationship.getObjectField();
+
+				_objectFieldLocalService.updateRequired(
+					relationshipObjectField.getId(), true);
```

Comment:
@G-Sa can we move the `_objectFieldLocalService.updateRequired` to be inside the `_objectRelationshipLocalService.updateEdge`?